components/TxConfirmationDialog.qml

```diff
@@ -0,0 +1,458 @@
+// MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+// THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+// THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import QtQuick 2.9
+import QtQuick.Controls 1.4
+import QtQuick.Layouts 1.1
+
+import "../components" as MoneroComponents
+import FontAwesome 1.0
+
+Rectangle {
+    id: root
+    x: parent.width/2 - root.width/2
+    y: parent.height/2 - root.height/2
+    z: parent.z + 1
+    // TODO: implement without hardcoding sizes
+    width: 580
+    height: 400
+    color: MoneroComponents.Style.blackTheme ? "black" : "white"
+    visible: false
+    radius: 10
+    border.color: MoneroComponents.Style.blackTheme ? Qt.rgba(255, 255, 255, 0.25) : Qt.rgba(0, 0, 0, 0.25)
+    border.width: 1
+
+    property var transactionAmount: ""
+    property var transactionAddress: ""
+    property var transactionDescription: ""
+    property var transactionFee: ""
+    property var transactionPriority: ""
+    property var transactionID: ""
+    property alias dialogTitle: dialogTitle
+    property alias errorText: errorText
+    property alias confirmButton: confirmButton
+    property alias backButton: backButton
+    property alias bottomText: bottomText
+    property alias buttons: buttons
+    property alias bottomTextAnimation: bottomTextAnimation
+
+    state: "default"
+    states: [
+        State {
+            // waiting for user action, show tx details + back and confirm buttons
+            name: "default";
+            when: errorText.text =="" && bottomText.text ==""
+            PropertyChanges { target: errorText; visible: false }
+            PropertyChanges { target: txAmountText; visible: root.transactionAmount !== "(all)" || (root.transactionAmount === "(all)" && currentWallet.isHwBacked() === true) }
+            PropertyChanges { target: txAmountBusyIndicator; visible: !txAmountText.visible }
+            PropertyChanges { target: txFiatAmountText; visible: txAmountText.visible && persistentSettings.fiatPriceEnabled && root.transactionAmount !== "(all)" }
+            PropertyChanges { target: txDetails; visible: true }
+            PropertyChanges { target: bottom; visible: true }
+            PropertyChanges { target: bottomMessage; visible: false }
+            PropertyChanges { target: buttons; visible: true }
+            PropertyChanges { target: backButton; visible: true; primary: false }
+            PropertyChanges { target: confirmButton; visible: true }
+            StateChangeScript { script: confirmButton.forceActiveFocus() }
```

Comment:
this breaks the hover animation :/ I wonder if this can be done in a different way

Reply:
Maybe we create an additional transparent button that is only used to receive the forced focus when the pop up opens?

Reply:
Will play around with it a bit, don’t change anything yet.

Reply:
done